lib/Controller/RoomController.php

```diff
@@ -971,6 +973,29 @@ public function renameRoom(string $roomName): DataResponse {
 		return new DataResponse();
 	}
 
+	/**
+	 * @PublicPage
+	 * @RequireModeratorParticipant
+	 *
+	 * @param string $description
+	 * @return DataResponse
+	 */
+	public function setDescription(string $description): DataResponse {
+		if ($this->room->getType() === Room::ONE_TO_ONE_CALL) {
+			return new DataResponse([], Http::STATUS_BAD_REQUEST);
+		}
+
+		if (mb_strlen($description) > Room::DESCRIPTION_MAXIMUM_LENGTH) {
+			return new DataResponse([], Http::STATUS_BAD_REQUEST);
+		}
+
+		$description = trim($description);
```

Comment:
trim before checking the length like in `lib/Command/Room/TRoomCommand.php`

Reply:
Also all this logic could be in `$this->room->setDescription` so you only need to do it in one place